lib/leak_tracker.dart

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
```

Comment:
Is this empty file necessary?

Reply:
It is a placeholder for now. Created it, to have it reviewed, as the file structure is also subject for review. 

Added comment.